anyway/parsers/news_flash.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+
+from . import twitter, rss_sites
+from .news_flash_db_adapter import init_db
+from .news_flash_classifiers import classify_rss, classify_tweets
+from .location_extraction import extract_geo_features
+
+# FIX: classifier should be chosen by source (screen name), so `twitter` should be `mda`
+news_flash_classifiers = {"ynet": classify_rss, "twitter": classify_tweets, "walla": classify_rss}
+
+
+def update_all_in_db(source=None, newsflash_id=None):
+    """
+    main function for newsflash updating.
+
+    Should be executed each time the classification or location-extraction are updated.
+    """
+    db = init_db()
+    if newsflash_id is not None:
+        newsflash_items = db.get_newsflash_by_id(newsflash_id)
+    elif source is not None:
+        newsflash_items = db.select_newsflash_where_source(source)
+    else:
+        newsflash_items = db.get_all_newsflash()
+
+    for newsflash in newsflash_items:
+        classify = news_flash_classifiers[newsflash.source]
+        newsflash.accident = classify(newsflash.description or newsflash.title)
+        if newsflash.accident:
+            extract_geo_features(db, newsflash)
+    db.commit()
+
+
+def scrape_extract_store_rss(site_name, db):
+    latest_date = db.get_latest_date_of_source(site_name)
+    for newsflash in rss_sites.scrape(site_name):
+        if newsflash.date <= latest_date:
+            break
+        newsflash.accident = classify_rss(newsflash.description or newsflash.title)
```

Comment:
Used to be `newsflash.title`. Perhaps this should be two calls.

Reply:
Perhaps the right thing to do is `'\n'.join([description, title])` either here or inside the classification.

Reply:
@elazarg sounds good, perhaps title and then description?
We need to understand how it influences the extraction of the location string. @Mano3 what do you think?

Reply:
Sorry, due to a bug this was only sent now.

Reply:
We still don't have a proper validation set on which to work. If you don't consider this as a bug, I'd better leave it the way it used to be.